packages/scene-composer/src/augmentations/components/three-fiber/anchor/AnchorWidget.tsx

```diff
@@ -124,11 +128,48 @@ export function AsyncLoadedAnchorWidget({
       WarningIconSvgString,
       ErrorIconSvgString,
       VideoIconSvgString,
+      CustomIconSvgString,
     ];
     return iconStrings.map((iconString, index) => {
+      if (keys[index] === 'Custom') {
+        const parser = new DOMParser();
+        const svgDocument = parser.parseFromString(iconString, 'image/svg+xml');
+        const svgRoot = svgDocument.documentElement;
+
+        const replaceFillAttribute = (element: Element) => {
+          const tagName = element.tagName.toLowerCase();
+          if (tagName === 'ellipse') {
+            if (chosenColor === undefined) {
+              return;
+            } else {
+              THREE.Cache.clear();
+              element.setAttribute('stroke', chosenColor);
+            }
+          }
+          if (tagName === 'circle') {
+            if (chosenColor === undefined) {
```

Comment:
nitpick: you could move the if chosenColor === undefined test to the outside since it does the same action for both tagName options.